grails-app/services/au/org/emii/portal/BulkDownloadService.groovy

```diff
@@ -145,4 +145,14 @@ class BulkDownloadService {
         zipStream.write bytes, 0, bytes.length
         zipStream.closeEntry()
     }
+
+    public static void copy(final InputStream input, final OutputStream output, final int bufferSize) throws IOException
```

Comment:
> IOUtils.copy function closes both streams when it finishes, which is not good for us.

I don't understand - this was the previous behaviour (?) but you're saying that it doesn't work?